#134491373 fix style issues and conflict with previously raised pull request

vonvick opened the pull request:
#### What does this PR do?
This PR fixes the violations found in Hound CI as a result of non-usage of ESlint
#### Description of Task to be completed?
Edit files that did not pass Hound CI test and make them pass. Afterwards create a pull request
#### How should this be manually tested?
This can be tested when reading the Hound CI report before merging
#### Any background context you want to provide?
No background context
#### What are the relevant pivotal tracker stories?
Edit files that did not pass Hound CI test and make them pass. Afterwards create a pull request
#### Screenshots (if appropriate)
#### Questions:
No questions

Commit:
style(style-formatting): edit styling for gulpfile

Edit the gulpfile to use the eslint style configuration

start #134491373
finishes #134491373

Commit:
style(style-formatting): fix indent for travis file

Edit the .travis.yml to follow same inden tation

start #134491373

Commit:
bug(merge-conflict): resolve merge conflict

Resolve merge conflict between my branch and the branch.

fixes #134491373

oagboola commented on gulpfile.js:
```diff
@@ -7,65 +9,52 @@ var browserSync = require('browser-sync');
 var filter = require('gulp-filter');
-require('dotenv').config();
-var port = process.env.PORT;
 
```
any reason for this empty line?

oagboola commented on gulpfile.js:
```diff
@@ -7,65 +9,52 @@ var browserSync = require('browser-sync');
+    .pipe(istanbul.hookRequire());
 });
 
 
```
empty line here too

Commit:
bug(fix-style-issues): fix style issues with eslint on HoundCI

Fic the issues that are raised by Hound CI when trying to create a pull request.

Fixes #134491373 Finishes #134491373

Commit:
bug(fix-houndci-issues): fix issues with hound violations on package.json

Fix issues with houndci on gulpfile.js

Fixes #134491373

oagboola commented:
you need to resolve the conflicts, can't merge until you do that

Commit:
bug(edit-files): edit files to fix ho

Commit:
bug(fix-houndci-issues): fix houndci issues when raising a pull request

edit gulpfile.js, eslintrc and package.json files to allow the use of 'require' in javascript files

fixes #134491373

Commit:
bug(merge-conflict): fix merge conflict from master branch

Finish #134491373

Commit:
bug(fix-require-issues): fix require issues with es6 in eslint

fixes #134491373

Commit:
bug(eslint-require): add global require to gulpfile.js

add global require to gulpfile.js to allow require to be used for importing modules

fixes #134491373

Commit:
bug(eslint-require): add global require to gulpfile.js

add global require to gulpfile.js to allow require to be used for importing modules

fixes #134491373
